app/classifier/tasks/survey/index.cjsx

```diff
@@ -48,7 +48,10 @@ module.exports = React.createClass
       {if @state.selectedChoiceID is ''
         <Chooser task={@props.task} filters={@state.filters} onFilter={@handleFilter} onChoose={@handleChoice} onRemove={@handleRemove} annotation={@props.annotation} focusedChoice={@state.focusedChoice} />
       else
-        <Choice task={@props.task} choiceID={@state.selectedChoiceID} onSwitch={@handleChoice} onCancel={@clearSelection} onConfirm={@handleAnnotation} />}
+        # @ is undefined within the scope of find
+        currentSelection = @state.selectedChoiceID
+        existingAnnotation = @props.annotation.value.find (value) -> value.choice is currentSelection || {}
```

Comment:
`undefined` is a valid value for `existingAnnotation` here (there might not be an annotation yet) so you don't need to set a default value of `{}`. The `Choice` component is responsible for setting a default value if the annotation doesn't exist.